common/djangoapps/course_action_state/managers.py

```diff
@@ -143,7 +143,7 @@ def failed(self, course_key):
         self.update_state(
             course_key=course_key,
             new_state=self.State.FAILED,
-            message=traceback.format_exc(),
+            message=traceback.format_exc()[-1000:],  # truncate to fit CHAR(1000)
```

Comment:
Can we declare a constant for this number in this file and then use that constant in both the test and here?